PROTOCOL.md

```diff
@@ -75,12 +80,23 @@ The client starts to considerer the keep alive message only upon the first recei
 
 This is a demonstration of client-server communication, in order to get a better understanding of the protocol flow:
 
+
 - Server configured with GraphQL schema (and `PubSub` implementation when using subscriptions)
 - Client creates a WebSocket instance using `SubscriptionsClient` object.
+
```

Comment:
Let's separate this part into example.MD or something like that as this is implementation related and not only protocol. this is a practical example of how this server works.